sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -251,20 +251,21 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
   }
 
   /**
-   * Get programs for a particular location.
-   * @param locationId id of the location that should have the
-   * offering associated with.
-   * @returns programs under the specified location.
+   * Get programs for a particular institution.
+   * @param institutionId id of the institution.
+   * @returns programs under the specified institution.
    */
-  async getPrograms(locationId: number): Promise<Partial<EducationProgram>[]> {
+  async getPrograms(
```

Comment:
Just to clarify, is this an existing method and does it have any dependency?

Reply:
No this is part of the previous bug and I did fixed it